app/search/dataone.py

```diff
@@ -148,9 +153,11 @@ def convert_result(self, result):
             spatial_coverage=result.pop('placeKey', None),
             author = result.pop('author',None),
             doi=doi,
+            boundingbox = {'south': result.pop('southBoundCoord',None), 'north':result.pop('northBoundCoord',None),'west': result.pop('westBoundCoord',None), 'east':result.pop('eastBoundCoord',None)},
```

Comment:
I think it'd be best to build the result geometry in this file and not have `boundingbox` be a thing in the SearchResult class, because the way this bounding box works is specific to DataONE, and the SearchResult class is meant to be general to any search result. You will likely need to move the test that you wrote to `test_dataone.py`. Thank you!